package.json

```diff
@@ -25,9 +25,10 @@
     "eslint-plugin-mocha": "4.9.0",
     "eslint-plugin-react": "7.0.1",
     "mocha": "3.4.2",
-    "moment": "2.18.1"
+    "moment": "2.18.1",
+    "pg": "6.2.3"
   },
-  "dependencies": {
+  "peerDependencies": {
     "pg": "6.2.3"
```

Comment:
I also suggest to not be so strict about the peer dependency. Use `^` or `~` here. IMO, `^`

Reply:
All other dependencies are *exact* ... @ThieryMichel What do you prefer ?

Reply:
yes, but there you're forcing users to use an exact version. This will be a PITA, I assure you